### ISSUE TITLE
Bootstrap 4 form-input error message in the wrong place

### ISSUE THREAD

**Issue by ezmemoar** [protonemedia/laravel-form-components#63]
When I tried to validate the input, `form-input` didn't show any error message. So, I checked up the HTML and it shows that the error message are placed outside the `input-group` class
```
<div class="input-group">
     <input/>
</div>

{!! $help ?? null !!}

@if($hasErrorAndShow($name))
     <x-form-errors :name="$name" />
@endif
```
which should be inside it, or more precisely placed below the input
```
<div class="input-group">
     <input/>

     {!! $help ?? null !!}

     @if($hasErrorAndShow($name))
          <x-form-errors :name="$name" />
     @endif
</div>
```

**Comment by pascalbaljet** [comment]
Fixed with #61

**Comment by pascalbaljet** [comment]
Fixed in v3.1.0.
